test/test_browser.py

```diff
@@ -773,7 +774,6 @@ def test_sdl_image(self):
           '-O2', '-lSDL', '-lGL',
           '--preload-file', dest, '-DSCREENSHOT_DIRNAME="' + dirname + '"', '-DSCREENSHOT_BASENAME="' + basename + '"', '--use-preload-plugins'
         ])
-      return
```

Comment:
This return was causing part of the test to be skipped, which I think was just a bug before.